example_configs/activemq.yml

```diff
@@ -0,0 +1,48 @@
+hostPort:
+username:
+password:
+lowercaseOutputLabelNames: true
+blacklistObjectNames:
+  - "org.apache.activemq:clientId=*,*"
+whitelistObjectNames:
+  - "java.lang:type=OperatingSystem"
+  - "java.lang:type=Threading"
+  - "org.apache.activemq:destinationType=Queue,*"
+  - "org.apache.activemq:destinationType=Topic,*"
+  - "org.apache.activemq:type=Broker,brokerName=*"
+  - "org.apache.activemq:type=Topic,brokerName=*"
+
+rules:
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*), destinationType=Queue, destinationName=(\S*)<>(\w+)
+  name: activemq_queue_$3
+  attrNameSnakeCase: true
+  labels:
+    broker: $1
+    destination: $2
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*), destinationType=Topic, destinationName=(\S*)<>(\w+)
+  name: activemq_topic_$3
+  attrNameSnakeCase: true
+  labels:
+    broker: $1
+    destination: $2
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>CurrentConnectionsCount
+  name: activemq_current_connections_count
```

Comment:
Gauges shouldn't end in _count

Reply:
I renamed to just `activemq_current_connections`